src/octoprint/plugins/softwareupdate/static/js/softwareupdate.js

```diff
@@ -834,13 +843,8 @@ $(function () {
                     "Be sure to read through any linked release notes, especially those for OctoPrint since they might contain important information you need to know <strong>before</strong> upgrading."
                 ) +
                 "</p>" +
-                "<p><strong>" +
-                gettext("This action may disrupt any ongoing print jobs.") +
-                "</strong></p>" +
                 "<p>" +
-                gettext(
-                    "Depending on your printer's controller and general setup, restarting OctoPrint may cause your printer to be reset."
-                ) +
```

Comment:
So, the reason for this back then when was the following scenario:

User has SD print ongoing, OctoPrint currently not connected via serial but physically. User clicks update. Update runs, server reboots. Server is set to connect to printer on startup. Server connects, printer reboots (it's still one of THOSE), boom, print kaputt.

So I'm not so sure right now this should be taken out completely here, even with a queue this COULD happen.

Reply:
Put back in, feel free to adjust the language to your liking.